crmsh/bootstrap.py

```diff
@@ -2596,19 +2645,30 @@ def remove_qdevice():
     if not confirm("Removing QDevice service and configuration from cluster: Are you sure?"):
         return
 
+    utils.check_all_nodes_reachable()
+    _context.qdevice_reload_policy = evaluate_qdevice_quorum_effect(QDEVICE_REMOVE)
+
     status("Disable corosync-qdevice.service")
     invoke("crm cluster run 'systemctl disable corosync-qdevice'")
-    status("Stopping corosync-qdevice.service")
-    invoke("crm cluster run 'systemctl stop corosync-qdevice'")
+    if _context.qdevice_reload_policy == QdevicePolicy.QDEVICE_RELOAD:
+        status("Stopping corosync-qdevice.service")
+        invoke("crm cluster run 'systemctl stop corosync-qdevice'")
```

Comment:
Probably we could move this into the later block:
```
    if _context.qdevice_reload_policy == QdevicePolicy.QDEVICE_RELOAD:
        invoke("crm cluster run 'crm corosync reload'")
```
and put it right before `crm corosync reload`?

Reply:
Actually I did this intentionally.
`Stopping qdevice service -> then remove configuration -> then reload`
If
`remove configuration-> stop service and reload`
corosync-qdevice will generate error logs since the configure already broken

Reply:
Alright, a bit surprised a stopping service cares about the configuration.